cmd/kube-controller-manager/app/batch.go

```diff
@@ -44,9 +40,6 @@ func startJobController(ctx ControllerContext) (http.Handler, bool, error) {
 }
 
 func startCronJobController(ctx ControllerContext) (http.Handler, bool, error) {
-	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v1", Resource: "cronjobs"}] {
```

Comment:
Help me find the cronjob conformance test?  it's GA, but I don't see any `framework.ConformanceIt` test for cronjobs.

Reply:
> Help me find the cronjob conformance test? it's GA, but I don't see any `framework.ConformanceIt` test for cronjobs.

nevermind, stale level.  I found it.